operator-common/src/main/java/io/strimzi/operator/common/ShutdownHook.java

```diff
@@ -29,19 +42,18 @@ public void run() {
         if (vertx != null) {
```

Comment:
We can do the null check in the constructor, so we don't need it here.

Reply:
Much better. Thanks.